README.md

```diff
@@ -1,5 +1,20 @@
 # EpicAdvice
 
+## Development Prerequisites: 
+Installed: Python 3.12, IDE of your choice
```

Comment:
I am using python 3.9, not sure if that will introduce incompatibility. but if the program can run in 3.12 then there will be no problem